docs/guides/mongodb/vault-integration/kmip-encryption/index.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Vault Integration
```

Comment:
`Encryption with Vault KMIP`

Reply:
![Screenshot from 2024-09-24 19-32-57](https://github.com/user-attachments/assets/a7f7f2b9-ced4-411c-8c4d-96b26801a1dc)
Sidebar gone ! fix this